### PR TITLE
Update table, and add new column and use TRANSACTION statement in PostgreSQL

### PR DESCRIPTION
**Pull request by fezzopro**
## PR Details 🏬 

- [x] Use database transactions
  - [x] Update database
  - [x] Update table
  - [x] Delete data in SQL
  - [x] Rollback Transactions
  - [x] Prepare complex queries that answer analytical questions

## Screenshots

image

image

image

image

image

image

image

image

image

image

image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -34,4 +34,60 @@ SELECT *
...
+	UPDATE animals SET species= 'unspecified';
+	SELECT * FROM animals;
+ROLLBACK TRANSACTION;
+	SELECT * FROM animals;
+
+BEGIN TRANSACTION;
+	UPDATE animals SET species= 'digimon' WHERE TRIM(name) like '%mon';
+	UPDATE animals SET species= 'pokemon' WHERE species IS NULL;
+	SELECT * FROM animals;
+ROLLBACK TRANSACTION;
```
Comment (Kamba56) [inline queries.sql:49]: - [ ] As requested by project requirements, you are supposed to commit transaction here not rollback. Kindly fix this please.

File: queries.sql [hunk 2]
```diff
@@ -34,4 +34,60 @@ SELECT *
...
+SELECT * FROM animals;
+
+BEGIN TRANSACTION;
+	DELETE FROM animals;
```
Comment (Kamba56) [inline queries.sql:53]: - [ ] Kindly add a check after deleting to confirm that all animals where deleted.
`SELECT * FROM animals;`

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @fezzopro, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great job adding more data to animals table!👏👏👏
- [x] Great job using transactions to carry out queries

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (fezzopro) [reply queries.sql]: Thank you for spotting this, I have committed the transaction at this point instead of rolling back.

File: queries.sql (hunk 2 above)
Comment (fezzopro) [reply queries.sql:53]: @Kamba56 
Thanks for the review, I have added the check statement on line number 54.

**Review by rotimiazeez — APPROVED** [review]
# Hi @fezzopro 👏🏽, 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Good job correcting your queries as suggested 👏🏾 
* All data are in the correct file 🤝🏽
* Descriptive PR title and summary 👌🏽
* Screenshot of results are present in the PR 👍🏼

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fezzopro** [comment]
@rotimiazeez Thank you for the review 🍾

**Pull request merged**